### PR TITLE
Cy 3049 add supervisord to all places in cloudify that still call to systemctl

### PR DESCRIPTION
**Pull request by mabuaisha**

### CODE REVIEW

File: cfy_manager/main.py [hunk 1]
```diff
@@ -523,6 +528,14 @@ def _are_components_configured():
     return os.path.isfile(INITIAL_CONFIGURE_FILE)
 
 
+def is_supervisord_service():
+    service_type = service._get_service_type()
+    # This only relevant for restarting services on VM that use supervisord
+    if service_type == 'supervisord':
+        return True
+    return False
```
Comment (tehasdf) [inline cfy_manager/main.py:536]:
```suggestion
def is_supervisord_service():
    return service._get_service_type() == 'supervisord'
```

File: packaging/files/usr/lib/systemd/system/supervisord.service [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Documentation=http://supervisord.org
+After=network.target
+
+[Service]
+ExecStart=/usr/bin/supervisord -n -c /etc/supervisord.conf
+ExecStop=/usr/bin/supervisorctl -c /etc/supervisord.conf shutdown
+ExecReload=/usr/bin/supervisorctl -c /etc/supervisord.conf reload
+KillMode=process
+Restart=on-failure
+RestartSec=42s
```
Comment (tehasdf) [inline packaging/files/usr/lib/systemd/system/supervisord.service:12]: 42?

Comment (mabuaisha) [reply packaging/files/usr/lib/systemd/system/supervisord.service:12]: This should be changed as I was testing, will update that one around. 10 or something

File: cfy_manager/main.py (hunk 1 above)
Comment (mabuaisha) [reply cfy_manager/main.py]: Done

File: packaging/files/usr/lib/systemd/system/supervisord.service (hunk 2 above)
Comment (mabuaisha) [reply packaging/files/usr/lib/systemd/system/supervisord.service]: Updated to 10s

**Pull request merged**